content/lessons/chapter-3/coop-2.tsx

```diff
@@ -175,7 +192,11 @@ export default function Coop2({ lang }) {
                   }
                 )}
               >
-                {PROFILES[i].value}
+                {i === PROFILES.length - 1
+                  ? profile.value
+                  : i === highestHashpowerProtagonist
+                  ? profile.value + 3
+                  : '0'}
```

Comment:
Not sure what is happening here, but I would recommend you do something like this instead:

```tsx
{i === PROFILES.length - 1 && profile.value}
{i === highestHashpowerProtagonist && profile.value + 3}
{i !== PROFILES.length - 1 && i !== highestHashpowerProtagnist && '0'}
```

While this probably takes more compute, I think its neglible if it makes it easier to parse than nested ternary operators.